crates/biome_js_analyze/tests/specs/correctness/noUnusedImports/invalid-import-namespace.ts.snap

```diff
@@ -51,7 +51,8 @@ invalid-import-namespace.ts:4:18 lint/correctness/noUnusedImports  FIXABLE  ━
     2 2 │   export type T1 = Ns1;
     3   │ - 
     4   │ - import·type·*·as·Ns2·from·""
-    5 3 │   export type T2 = Ns2;
+      3 │ + 
+    5 4 │   export type T2 = Ns2;
```

Comment:
This seems to be a regression

Reply:
Actually, this is a fix.
If you look at the entire context, here is the suggestion:

```
  i Safe fix: Remove the unused imports.
  
    1 1 │   import * as Ns1 from ""
    2 2 │   export type T1 = Ns1;
    3   │ - 
    4   │ - import·type·*·as·Ns2·from·""
      3 │ + 
    5 4 │   export type T2 = Ns2;
```

As you can notice, previously we were removing the blank line above the removed import.
Now we keep it :)

The diff is not so good. I could try to improve it by moving the trivia at the previous sibling in priority instead of the next one.
I chose the next one because it seems more conventional of associating these trivia as leading trivia.

Reply:
I applied the change. The diff should be better :)

Reply:
Totally missed that! Great catch